testing/trino-server-dev/etc/config.properties

```diff
@@ -52,6 +52,7 @@ plugin.bundles=\
   ../../plugin/trino-thrift/pom.xml, \
   ../../plugin/trino-tpcds/pom.xml, \
   ../../plugin/trino-google-sheets/pom.xml, \
+  ../../plugin/trino-bigquery/pom.xml, \
```

Comment:
No need to change this class. You can use BigQueryQueryRunner instead.

Reply:
It increases bootstrap time and we don't have catalog config property file in `catalog` directory. I would recommend reverting this change.

Reply:
Removed